src/Razor/src/Microsoft.VisualStudio.RazorExtension/AssemblyCodeBases.cs

```diff
@@ -5,11 +5,12 @@
 
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.LanguageServer.Protocol.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.VisualStudio.LanguageServer.Protocol.Extensions.dll")]
+[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\MediatR.dll")]
```

Comment:
This'll probably cause regressions on insert. 😞

Reply:
Went from binding redirect -> codebase. This one should be ok. Also, all these are only ever loaded in LSP powered scenarios which isn't DDRITs tested currently so I wouldn't imagine it'd be a problem